backend/test_chinese_website_parsing.py

```
@@ -0,0 +1,308 @@
+"""
+import os
+import tempfile
+import uuid
+
+from sqlmodel import Session
+
+from app.core.db import engine
+from app.models.content import ContentItem
+
+
+def test_chinese_website_parsing():
```

Pylint (3.3.7)

[refactor] Too many local variables (50/15)

(R0914)